src/templates/mapOverlay.ejs

```diff
@@ -25,6 +25,9 @@ Parameters:
     - visible : Required : String
         A knockout data-bind for visibility when true
 
+    - moveToTown : Optional : String
```

Comment:
Not really a "moveToTown". That kinda indicates a clickaction. Let's call it something like "connectedToTown" or something